2025-10-18 02:56:31 - app - ERROR - [logger.py:104] - Unexpected error for http://drugybsencwyzc6sw6nzecuoavw5cvmqq2dhyojqaghfuve2dmzs4jqd.onion/: Host unreachable
2025-10-18 02:57:25 - app - ERROR - [logger.py:104] - Unexpected error for https://onioj4brbi42axgtklgj5nqbq2p2txnjsjdq6eowkznaykddpezuvxid.onion/: Host unreachable
2025-10-18 02:59:40 - app - ERROR - [logger.py:104] - Unexpected error for https://r7lscyg5lncab4gm77ldshetqoutj3btq53aw44ixnz3iinc3oaz7qqd.onion/: Connection refused by destination host
2025-10-18 03:02:47 - app - ERROR - [logger.py:104] - Unexpected error for http://epato7h7ttmbxmyhwnr5dmufaskkuvwuasdhia3haqg3526fftg2jbad.onion/: Host unreachable
2025-10-18 03:02:54 - app - ERROR - [logger.py:104] - Unexpected error for http://drugidjjstv6lklmqeqd7eypjjtc24niaskpivrwa5wmb3vwd2ndwgyd.onion/: Host unreachable
